### PR TITLE
Support disable_notify and message options in add-devices command

### PR DESCRIPTION
**Pull request by anoworl**
In addition to dg deploy command, I supported disable_notify and message options in dg add-devices command.

## Checklist
- [x] dg add-devices --disable_notify
    - No email sent
- [x] dg add-devices --message hogehoge
    - Release note changed
- [x] dg add-devices -h

### CODE REVIEW

File: lib/deploygate/command_builder.rb [hunk 1]
```diff
@@ -86,12 +86,14 @@ def run
       command ADD_DEVICES do |c|
         c.syntax = 'dg add-devices'
         c.description = I18n.t('command_builder.add_devices.description')
+        c.option '--message STRING', String, I18n.t('command_builder.add_devices.message')
```
Comment (jmatsu) [inline lib/deploygate/command_builder.rb:89]: It looks better for me to use `command_builder.deploy.message` to represent *reuse*. What do you think? @anoworl 

ref: https://github.com/DeployGate/deploygate-cli/pull/305/files#diff-79f3cff459436871e580bb0a25dc3de5b4c3830f2bb92915d7d811ad65574d85R94

File: lib/deploygate/command_builder.rb [hunk 2]
```diff
@@ -86,12 +86,14 @@ def run
...
         c.option '--user STRING', String, I18n.t('command_builder.add_devices.user')
         c.option '--udid STRING', String, I18n.t('command_builder.add_devices.udid')
         c.option '--device-name STRING', String, I18n.t('command_builder.add_devices.device_name')
         c.option '--distribution-key STRING', String, I18n.t('command_builder.add_devices.distribution_key')
         c.option '--configuration STRING', String, I18n.t('command_builder.deploy.configuration')
         c.option '--server', I18n.t('command_builder.add_devices.server.description')
+        c.option '--disable_notify', I18n.t('command_builder.add_devices.disable_notify')
```
Comment (jmatsu) [inline lib/deploygate/command_builder.rb:96]: ditto

**Review by jmatsu — CHANGES_REQUESTED** [review]

File: lib/deploygate/command_builder.rb (hunk 1 above)
Comment (anoworl) [reply lib/deploygate/command_builder.rb:89]: Thank you. True. I will change it.

Comment (anoworl) [reply lib/deploygate/command_builder.rb]: I did! https://github.com/DeployGate/deploygate-cli/pull/305/commits/21cd2798026465465a46f499064ec078b2553be2

And checked "dg add-devices -h" outputs.

**Review by anoworl — COMMENTED** [review]
Thank you! I reflected and replied the reviews.

File: lib/deploygate/command_builder.rb [hunk 3]
```diff
@@ -86,13 +86,15 @@ def run
       command ADD_DEVICES do |c|
         c.syntax = 'dg add-devices'
         c.description = I18n.t('command_builder.add_devices.description')
-        c.option '--user STRING', String, I18n.t('command_builder.add_devices.user')
+        c.option '--message STRING', String, I18n.t('command_builder.deploy.message')
+        c.option '--user STRING', String, I18n.t('command_builder.deploy.user')
```
Comment (jmatsu) [inline lib/deploygate/command_builder.rb:90]: Great! 💯

**Review by jmatsu — APPROVED** [review]
LGTM

**Comment by anoworl** [comment]
Thank you!

**Pull request merged**